v4/source/opengraph.yaml

```diff
@@ -0,0 +1,16 @@
+  /opengraph:
+    post:
+      tags:
+        - opengraph
+      summary: Get Open Graph Metadata for a specif URL
+      description: |
+        Get Open Graph Metadata for a specif URL.
```

Comment:
And this to `Use the Open Graph protocol to get some generic metadata about a URL. Used for creating link previews.`